isis/src/qisis/objs/SortFilterProxyModel/SortFilterProxyModel.cpp

```diff
@@ -90,6 +90,39 @@ namespace Isis {
  }
 
 
+  QVariant SortFilterProxyModel::data(const QModelIndex &index, int role) const {
+    if (!index.isValid()) return QVariant();
+
```

Comment:
something I noticed when running this, if we want the proxy model to show the icons (the little hot-air balloon icons next to images on the project tree), we weill need to add a check for the _Qt::DecorationRole_ and call the QSortFilterProxyModel::data function (like we do for the Qt::DisplayRole)